app/models/token/expirable_token.rb

```diff
@@ -64,30 +64,34 @@ def valid_plaintext?(input)
       end
 
       def expired?
-        expires_on.nil? || Time.now > expires_on
+        expires_on.nil? || Time.current > expires_on
       end
 
-      def validity_time
-        self.class.validity_time
-      end
+      delegate :validity_time, to: :class
 
       ##
       # Set the expiration column
       def set_expiration_time
-        self.expires_on = Time.now + validity_time
+        self.expires_on = Time.current + validity_time
```

Comment:
Similar to my proposal above, but maybe a bit more controversial in terms of readability:

```suggestion
        self.expires_on = validity_time.from_now
```

It reads a bit nicer without a variable in between, e.g.

```ruby
5.days.from_now
```

Reply:
I agree 👍